shenyu-examples/shenyu-examples-dubbo/shenyu-examples-apache-dubbo-service-annotation/src/main/java/org/apache/shenyu/examples/apache/dubbo/service/annotation/impl/DubboMultiParamServiceImpl.java

```diff
@@ -32,7 +32,7 @@
 /**
  * The type Dubbo multi param service.
  */
-@DubboService
+@ShenyuDubboService("demo")
```

Comment:
rollback this

Reply:
Why did not this be rollbacked?  It seems that Dubbo only resolves the direct annotation, not composite annotation. (#4031)

Reply:
> Why was this not rollbacked? It seems that Dubbo only resolves the direct annotation, not composite annotation. (#4031)

please check the `master` branch, it has benn rollbacked.